lib/remap-lang/src/parser.rs

```diff
@@ -688,10 +703,7 @@ mod tests {
             ("..", vec![" 1:2\n", "= expected path segment"], Ok(vec![])),
             (
                 ". bar",
-                vec![
-                    " 1:3\n",
-                    "= expected assignment, if-statement, query, operator, or block",
-                ],
+                vec![" 1:6\n", "= unknown parsing error"],
```

Comment:
These will be improved once compiler-error messages are being tackled (planned for next week).